src/client.cpp

```diff
@@ -527,8 +527,8 @@ ProcessResponseEnum HttpClient::ProcessResponse(Value& result, bool notification
 bool TcpClient::GenerateHeader()
 {
     log_trace();
-    header_ << request_.Length() << ":";
-    log_debug("Request length=" << request_.Length());
+    header_ << static_cast<uint64_t>(request_.Length()) << ":";
```

Comment:
Trying to see if there is a cleaner way to do this.  Searching around, it looks like OS X defines the following.

    typedef unsigned int __uint32_t;
    typedef unsigned long long __uint64_t;
    typedef long unsigned int __darwin_size_t;
    typedef __darwin_size_t size_t;

The compiler doesn't think "long unsigned int" is exactly the same as "unsigned long long" or "unsigned int".
I'd rather avoid the casting inline.  Maybe we overload operator<< with something for size_t

stream.h

    #ifdef __APPLE__
    ANYRPC_API anyrpc::Stream& operator<<(anyrpc::Stream& os, size_t st);
    #endif

stream.cpp

    #ifdef __APPLE__
    anyrpc::Stream& operator<<(anyrpc::Stream& os, size_t st)
    {
        std::stringstream buffer;
        buffer << st;
        os.Put(buffer.str());
        return os;
    }
    #endif

You might need to cast with `buffer << st` depending on what types std::stringstream defines.  It needs to only be defined for `__APPLE__` to avoid a redefinition error on other platforms.